data_preparation/crop_statistics_AUS/README.md

```diff
@@ -0,0 +1,92 @@
+# Australia's Regional Wheat Yield
+
+## Short description
+
+The analysis-ready data of wheat yield for Australian's sub-state time series was derived from Production (t) and Area (ha) from the Australian Bureau of Agricultural and Resource Economics and Sciences (ABARES). See .R.
+
+ABARES maintains detailed spatial datasets related to Australia’s land and forests. These datasets are also available to use or view via spatial web services (also called web mapping services). Users can interact with spatial web services using: Geographic information systems (GIS) software installed on your computer & Online mapping applications (via a recently updated web browser) For information about ABARES other data holdings see ABARES latest data releases (https://www.agriculture.gov.au/abares/data) and the ABARES data catalogue (https://data.agriculture.gov.au/geonetwork/srv/eng/catalog.search;jsessionid=D688ADA3B12D94651DAA85ABC3639CC1#/home). Other Department of Agriculture, Water and Environment web services are also available (http://www.environment.gov.au/about-us/environmental-information-data/web-services).
+
+This data was download manually by navigating the website until getting this file: https://www.agriculture.gov.au/sites/default/files/documents/fdp-beta-regional-historical.csv
+
+## Link
+https://www.agriculture.gov.au/abares/data/farm-data-portal
+
+## Publisher
+ABARES
+
+## Dataset owner
+Commonweath of Australia
+
+## Data card author
+Richetti, Jonathan
+
+## Dataset overview
+**Crops**: Wheat
+
+**Variables** [unit]: t / ha

```

Comment:
please here the variables available in the database and their unit e.g. yield [t/ha], production [t], area sown [ha]. The R script shows the database also contains production and area sown.